velox/functions/lib/aggregates/SumAggregateBase.h

```diff
@@ -16,22 +16,22 @@
 #pragma once
 
 #include "velox/expression/FunctionSignature.h"
+#include "velox/functions/lib/CheckedArithmeticImpl.h"
 #include "velox/functions/lib/aggregates/DecimalAggregate.h"
 #include "velox/functions/lib/aggregates/SimpleNumericAggregate.h"
-#include "velox/functions/prestosql/CheckedArithmeticImpl.h"
 
-using namespace facebook::velox::functions::aggregate;
+using namespace facebook::velox::aggregate;
```

Comment:
do not use "using" in header file; looks like this is a pre-existing issue; would you submit a separate PR to fix that, than rebase this one?

Reply:
Hi @mbasmanova, after moving SumAggregateBase.h to `lib`, only SumHook came from another namespace, so I directly removed the using namespace in this PR and modified [SumHook](https://github.com/facebookincubator/velox/pull/6889/files#diff-3d23b55eb5bd17dafffdc46c9674a8c9968a0b26c2d411829d564574d03d549cR132) to use the complete namespace.